src/main/resources/restService/token/defaultToken.json

```diff
@@ -3,5 +3,10 @@
     "TestController": {
       "token": "Hallo"
     }
+  },
+  "pid": {
```

Comment:
I would say, this should be the plugin ID?

Reply:
This can be removed, since the SecondTestController has to have the same pluginId as TestController.